src/party_menu.c

```diff
@@ -1788,25 +1767,24 @@ u8 GetMonAilment(struct Pokemon *mon)
     return AILMENT_NONE;
 }
 
+#define minigameBitflag data[0]
```

Comment:
I think we usually prefix with t for these kinds of definitions, e.g. `tMingameBitflag`

Reply:
I assumed the convention was to use `t` prefix for task data (just like the `s` prefix for sprite data)?

Reply:
Yeah `t` is meant to be short for `task`. We don't have any prefix for generic struct data arrays, this is fine with me.